workers/generic-worker/build.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
```

Comment:
Is this script definitely compatible with non-bash shells?
Just curious what the reason was for this line change.

Reply:
Alpine linux doesn't have bash installed by default and builds the generic worker image just fine while running this script. :)